ComputerVision/SDK/ExtractText/Program.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Microsoft.Azure.CognitiveServices.Vision.ComputerVision;
+using Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Models;
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace ExtractText
+{
+    class Program
+    {
+        // subscriptionKey = "0123456789abcdef0123456789ABCDEF"
+        private const string subscriptionKey = "0123456789abcdef0123456789ABCDEF";
+
+        // For printed text, change to TextRecognitionMode.Printed
+        private const TextRecognitionMode textRecognitionMode = TextRecognitionMode.Handwritten;
+
+        private const string remoteImageUrl ="https://upload.wikimedia.org/wikipedia/commons/thumb/d/dd/Cursive_Writing_on_Notebook_paper.jpg/800px-Cursive_Writing_on_Notebook_paper.jpg";
+
+        private const int numberOfCharsInOperationId = 36;
+
+        static void Main(string[] args)
+        {
+            ComputerVisionClient computerVision = new ComputerVisionClient(new ApiKeyServiceClientCredentials(subscriptionKey),
+                new System.Net.Http.DelegatingHandler[] { });
+
+            // You must use the same region as you used to get your subscription
+            // keys. For example, if you got your subscription keys from westus,
+            // replace "westcentralus" with "westus".
+            //
+            // Free trial subscription keys are generated in the westcentralus
+            // region. If you use a free trial subscription key, you shouldn't
+            // need to change the region.
+
+            // Specify the Azure region
+            computerVision.Endpoint = "https://westus.api.cognitive.microsoft.com";
+            // localImagePath = @"C:\Documents\LocalImage.jpg"
+            string localImagePath = Directory.GetCurrentDirectory() + @"../../../../../../Images\sample2.png";
+
+
+            Console.WriteLine("Images being analyzed ...");
+            var t1 = ExtractRemoteTextAsync(computerVision, remoteImageUrl);
+            var t2 = ExtractLocalTextAsync(computerVision, localImagePath);
+
+            Task.WhenAll(t1, t2).Wait(5000);
+            Console.WriteLine("Press ENTER to exit");
+            Console.ReadLine();
+        }
+
+        // Read text from a remote image
+        private static async Task ExtractRemoteTextAsync(ComputerVisionClient computerVision, string imageUrl)
```

Comment:
ExtractTextRemoteAsync so we are consistent, and ExtractTextLocalAsync for the other case